### DISCUSSION TITLE
is this an issue?

### DISCUSSION THREAD

**Discussion by Tigertye** [CorsixTH/CorsixTH#2239] [General Diagnostics] [answered]
This happened in my last game and it's now happened in my new game. The doctors name lists as %s. Golde . It's a different doctors name this time but still has the %s. I've never seen this before v66.

Is this a real issue? 
Do I need to avoid hiring staff with names like this? 
Would they cause a problem with my game if I do?

**Comment by lewri** [comment, marked as answer]
Could well be a bug, if there's an issue I'd report it in issues: https://github.com/CorsixTH/CorsixTH/issues/new/choose

**Reply by Tigertye** [reply]
ok then. I'll hire one and keep an eye on him.

**Comment by lewri** [comment]
@tobylane any chance this could be to do with the staff first initial changes?

**Reply by tobylane** [reply]
Could be. Let's see how to recreate it.
